query/stats.go

```diff
@@ -57,97 +57,76 @@ func (s *Stat) reset() *Stat {
 
 // statGroup collects simple streaming statistics.
 type statGroup struct {
-	min    float64
-	max    float64
-	mean   float64
+	latencyHDRHistogram *hdrhistogram.Histogram
 	sum    float64
-	values []float64
-
-	// used for stddev calculations
-	m      float64
-	s      float64
-	stdDev float64
-
 	count int64
 }
 
 // newStatGroup returns a new StatGroup with an initial size
 func newStatGroup(size uint64) *statGroup {
+	// This latency Histogram could be used to track and analyze the counts of
```

Comment:
What happens for queries that take longer than 30s?

Reply:
it will return an error "value %d is too large to be recorded". will increase the limit to 60min since the overhead is not that much.